src/compute/src/sink/refresh.rs

```diff
@@ -115,7 +115,7 @@ where
                             // We can only get here if we see the frontier advancing to a time after
                             // the last refresh, but not empty, which would be a bug somewhere in
                             // the `until` handling.
-                            soft_panic_or_log!("frontier advancements after the `until` should be suppressed");
+                            /////////////////soft_panic_or_log!("frontier advancements after the `until` should be suppressed");
```

Comment:
Why is this commented out?

Reply:
See here: https://github.com/MaterializeInc/materialize/pull/24443#issuecomment-1896468096

Reply:
I've remove this soft_panic in an other PR: https://github.com/MaterializeInc/materialize/pull/24582